packages/test/test-version-utils/src/compatUtils.ts

```diff
@@ -52,98 +46,37 @@ import { getRequestedVersion } from "./versionUtils.js";
 export const TestDataObjectType = "@fluid-example/test-dataStore";
 
 /**
- * This function modifies container runtime options according to a version of runtime used.
- * If a version of runtime does not support some options, they are removed.
- * If a version runtime supports some options, such options are enabled to increase a chance of
- * hitting feature set controlled by such options, and thus increase chances of finding product bugs.
+ * Determines the minimumVersionForCollab that should be used for cross-client compatibility scenarios.
+ * We will use the lesser of the containerRuntimeVersion and containerRuntimeForLoadingVersion
```

Comment:
It's not apparent with `containerRuntimeForLoadingVersion` is. I would explain the parameters more in the comment. Maybe even give an example to make it clear. These are complicated concepts so the easier we can make it to understand, the better.